### ISSUE TITLE
personal note (stalker)

### ISSUE THREAD

**Issue by b0rah** [b0rah/cocoagaming#7]
I will be taking a shot at revamping Stalker to fill Phantom's old niche
Phantom used to be a terrifying stalker that would follow you silently
I will balance stalker around this
Its presence is felt, not heard, small flickers in the woods you might barely miss
While standing still it will have infinite invis for starters
In terms of other mechanics it'll need a lot of tweaking and testing
But it will  be a true stalker, a fleeting shadow
in other words he'll troll you
But if he gets found out, unlike phantom, he's FUCKED.
Phantoms could just brainlessly noclip
You'll have to evade and try and survive an enemy that's much stronger in direct combat
Play dirty.
You're a stalker, not a warrior. Use whatever means necessary.
That's the general idea, I have a lot I need to finish so it might be a while before stalker sees the light of day
But it'll get one more shot~
